packages/storefront-events-collector/src/handlers/product/addToCart.ts

```diff
@@ -10,7 +10,7 @@ const handler = (event: Event): void => {
         if(item.product.sku === productContext.sku){
             productCtx = createProductCtx(productContext);
         } else {
-            productCtx = createProductCtx(item.product)
+            productCtx = createProductFromCartItem(item)
```

Comment:
this is missing a semi-colon which in itself is not the end of the world but i don't understand why the linter didn't run here pre-commit... thoughts?

Reply:
I don't know if linter is set on this project.